### PR TITLE
13 create dto for student model

### PR DESCRIPTION
**Pull request by mrIminjonov**

### CODE REVIEW

File: TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace TestSchool.Models.Dtos
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/AddressRequestDto.cs]: Gofurjon hafa boma lekin bu joyda sen studentni branchiga yana adressni ham yuklab qoyibsan adresing ozi 14-barnchda edi shunchun sen buni ochirishing kk !

File: TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace TestSchool.Models.Dtos
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/AddressResponseDto.cs]: Buni ham ochirishing kerak

File: TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿namespace TestSchool.Models.Dtos
+{
+    public class StudentRequestDto
+    {
+        public int StudentId { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs]: StudentId kerak emas requestda

File: TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set;}
+
+        public string Email { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs:10]: Email ortqicha

File: TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+        public string Phone { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs:12]: Phone ham kerak emas

File: TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+﻿namespace TestSchool.Models.Dtos
+{
+    public class StudentResponseDto
+    {
+        public int StudentId { get; set; }
+
+        public int StudentName { get; set; }
```
Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs:7]: StudentLastName  qoshish kerak

Comment (isfandiyoro5) [inline TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs:7]: FirstName yoz nameni orniga 
lekin hafa bop ketmagin a  buni bitta menmas boshqalar bn tekshirdik !
agar StudentName yozmoqchi boseng uni avtomapperda qilish kk lekin uni bekzod aka qiladi

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

File: TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs (hunk 4 above)
Comment (JmlKhon) [reply TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs:10]: email bolishi kerak requestda @isfandiyoro5

File: TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs (hunk 5 above)
Comment (JmlKhon) [reply TestSchoolWebApi/Models/Dtos/StudentRequestDto.cs:12]: Phone ham qolishi kerak

File: TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs (hunk 6 above)
Comment (JmlKhon) [reply TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs:7]: response ga lastname bilan firstname ni qoshib StudentName qilsak boladi

Comment (JmlKhon) [reply TestSchoolWebApi/Models/Dtos/StudentResponseDto.cs:7]: StudentName  qoladi

**Review by isfandiyoro5 — DISMISSED** [review]

**Pull request closed without merging**